lib/config/presets/internal/replacements.ts

```diff
@@ -86,6 +87,17 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'material-ui-to-mui': {
+    description: 'the @material-ui/core monorepo was renamed to @mui/material',
+    packageRules: [
+      {
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/core'],
+        replacementName: '@mui/material',
+        replacementVersion: '5.0.0',
```

Comment:
It looks like `@matrial-ui/core` was updated to 4.x and then it switched to `@mui/material`. Should we at least check for current version 4.x, or propose an upgrade regardless?

Reply:
they have renamed a bunch of packages, not only 
@matrial-ui/core -> @mui/material
but also
@matrial-ui/icons -> @mui/icons-material
and others